base/iostream.jl

```diff
@@ -229,6 +229,17 @@ function readline(s::IOStream)
     ccall(:jl_readuntil, Ref{String}, (Ptr{Void}, UInt8, UInt8), s.ios, '\n', 1)
 end
 
+"""
+    readuntil_string(s::IO, delim::UInt8)
+
+Like `readuntil(s, delim)`, but returns a `String` rather than
+a `Vector{UInt8}`.
+"""
+readuntil_string(s::IO, delim::UInt8) = String(readuntil(s, delim))
+function readuntil_string(s::IOStream, delim::UInt8)
```

Comment:
Would it be better to make this `readuntil(s::IOStream, delim::UInt8)` instead of giving this a new name? Also other documented methods using readuntil return a string. 

I would rather rename the old `readuntil` to `readuntil_bytes` since it's not part of documented API anyway.